commit 30000261a201fc3dc7dbcc77d7c1873f589ee876
Author: Lukas Wunner <lukas@wunner.de>
Date:   Sat Jan 19 08:42:40 2019 +0100

    bcm2835-mmc: Deduplicate reset of driver data on remove
    
    The BCM2835 MMC host driver sets the device's driver data pointer to
    NULL on ->remove() even though the driver core subsequently does the
    same in __device_release_driver().  Drop the duplicate assignment.
    
    Signed-off-by: Lukas Wunner <lukas@wunner.de>
    Cc: Frank Pavlic <f.pavlic@kunbus.de>

diff --git a/drivers/mmc/host/bcm2835-mmc.c b/drivers/mmc/host/bcm2835-mmc.c
index 46ea11d5ed2e..68daa59d313b 100644
--- a/drivers/mmc/host/bcm2835-mmc.c
+++ b/drivers/mmc/host/bcm2835-mmc.c
@@ -1561,7 +1561,6 @@ static int bcm2835_mmc_remove(struct platform_device *pdev)
 		dma_release_channel(host->dma_chan_rxtx);
 
 	mmc_free_host(host->mmc);
-	platform_set_drvdata(pdev, NULL);
 
 	return 0;
 }
diff -uN a/1.txt b/1.txt
--- a/dummy/rpi_1391_30000261a201fc3dc7dbcc77d7c1873f589ee876.txt	1970-01-01 00:00:00.000000000 +0000
+++ b/dummy/rpi_1391_30000261a201fc3dc7dbcc77d7c1873f589ee876.txt	2013-12-23 04:07:40.000000000 +0000
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
